libraries/EEPROM/EEPROM.cpp

```diff
@@ -88,7 +88,7 @@ void EEPROMClass::end() {
 
 
 uint8_t EEPROMClass::read(int const address) {
-  if (address < 0 || (size_t)address >= _size)
+  if (address < 0 || ((size_t)address >= _size)) //Added brackets second part of condition for more readability
```

Comment:
If you know the operator precedence, then it's obvious. However, i don't mind the parentheses. Please drop the comment though, it explains your intent to us, but it's not helpful for the code.

Reply:
I do appreciate your respond, devyte. I didn't even think about that you will notice my little correction request on large project like this.  This is my first pull-request operation. I'm suppose to do pull-request job for my homework in open source project. I really interested Arduino and its modules. That i could be part of this project. I don't know culture of github yet. But i'll get used to it.